org.eclipse.xtext.testlanguages/src/org/eclipse/xtext/testlanguages/noJdt/NoJdtTestLanguage.mwe2

```diff
@@ -27,9 +27,9 @@ XtextGeneratorLanguage {
 
 	// The antlr parser generator fragment.
 	fragment = parser.antlr.XtextAntlrGeneratorFragment2 auto-inject {
-	//  options = {
-	//      backtrack = true
-	//  }
+	  options = {
+	     k = 1
```

Comment:
i hope i did the math correct. that do you think @szarnekow

Reply:
Antlr agrees.

Reply:
how/where can i see this?

Reply:
The generated code didn't change apart from line numbers in comments - no other LA related changes.